data_steward/tools/generate_unioned_ehr_dataset.sh

```diff
@@ -104,6 +136,36 @@ bq rm -f ${unioned_ehr_dataset}._mapping_ipmc_nu_procedure_occurrence
 bq rm -f ${unioned_ehr_dataset}._mapping_ipmc_nu_specimen
 bq rm -f ${unioned_ehr_dataset}._mapping_ipmc_nu_visit_occurrence
 
+# Run cleaning rules on unioned_ehr_dataset
+# create an intermediary table to apply cleaning rules on
+bq mk --dataset --description "intermediary dataset to apply cleaning rules on ${unioned_ehr_dataset_backup}" --label "phase:staging" --label "release_tag:${dataset_release_tag}" --label "de_identified:false" ${app_id}:${unioned_ehr_dataset_staging}
+
+"${TOOLS_DIR}/table_copy.sh" --source_app_id ${app_id} --target_app_id ${app_id} --source_dataset ${unioned_ehr_dataset_backup} --target_dataset ${unioned_ehr_dataset_staging}
+
+export UNIONED_DATASET_ID="${unioned_ehr_dataset_staging}"
+export BIGQUERY_DATASET_ID="${unioned_ehr_dataset_staging}"
+data_stage='unioned'
+
+# Remove de-activated participants
+if [ ${deactivation} == 'true' ];
+then
```

Comment:
I think we should always want to remove deactivated participants.  Why the conditional?

If we decide to always remove deactivated participants, we can get rid of the `deactivation` script argument.